src/main/kotlin/ru/mipt/npm/gradle/KScienceCommonPlugin.kt

```diff
@@ -46,14 +47,12 @@ open class KScienceCommonPlugin : Plugin<Project> {
                 }
             }
 
-            extensions.findByType<JavaPluginExtension>()?.apply {
+            configure<JavaPluginExtension> {
```

Comment:
This will fail if the extension does not exist. This construct is used for a reason

Reply:
Got it.